dashboard/db/migrate/20170929102458_add_details_to_regional_partner.rb

```diff
@@ -0,0 +1,43 @@
+class AddDetailsToRegionalPartner < ActiveRecord::Migration[5.0]
+  def up
+    add_column :regional_partners, :attention, :string
+    add_column :regional_partners, :street, :string
+    add_column :regional_partners, :apartment_or_suite, :string
+    add_column :regional_partners, :city, :string
+    add_column :regional_partners, :state, :string
+    add_column :regional_partners, :zip_code, :string
+    add_column :regional_partners, :phone_number, :string
+    add_column :regional_partners, :notes, :text
+    add_and_backfill_timestamps_for(RegionalPartner)
+  end
+
+  def down
+    remove_column :regional_partners, :attention
+    remove_column :regional_partners, :street
+    remove_column :regional_partners, :apartment_or_suite
+    remove_column :regional_partners, :city
+    remove_column :regional_partners, :state
+    remove_column :regional_partners, :zip_code
+    remove_column :regional_partners, :phone_number
+    remove_column :regional_partners, :notes
+    remove_timestamps_for(RegionalPartner)
+  end
```

Comment:
Why separate up and down migrations? It seems the only asymmetric part is the backfill timestamps.

Why not make this a `change` migration with a [reversible](http://guides.rubyonrails.org/active_record_migrations.html#using-reversible) block for the timestamps? Then you wouldn't have to have explicit `remove_column` statements for each new column in the down migration.